tests/core/test_surface.py

```diff
@@ -846,3 +846,41 @@ def test_miller_index_from_sites(self):
         s3 = np.array([1.1595, 0.66943764, 0.9065])
         hkl = miller_index_from_sites(matrix, [s1, s2, s3])
         assert hkl == (2, -1, 0)
+
+class TestCoherentInterfaceBuilder(unittest.TestCase):
```

Comment:
Thanks for adding these test, but perhaps they should go into `pymatgen/tests/analysis/interfaces/test_coherent_interface.py`

Reply:
OK

Reply:
Thanks a ton for all those changes!